app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UserController < ApplicationController
```

Comment:
[OPTIONAL BUT RECOMMENTED] Please replace `UserController` with `UsersController` because your controller name is `users_controller` not `user_controller` so kindly fix it otherwise it will give you error in future :slightly_smiling_face: